src/Views/AppInfoView.vala

```diff
@@ -85,12 +84,10 @@ namespace AppCenter.Views {
                 screenshot_stack.add (app_screenshot_not_found);
             }
 
-            package_name = new Gtk.Label (null);
-            package_name.margin_top = 12;
             package_name.selectable = true;
             package_name.xalign = 0;
             package_name.get_style_context ().add_class (Granite.STYLE_CLASS_H1_LABEL);
-            package_name.valign = Gtk.Align.CENTER;
+            package_name.valign = Gtk.Align.END;
```

Comment:
Seems this is the change that causes the misalignment of the package name and its version.

Reply:
I'm thinking we should group the name and version together so that they're guaranteed to be on the same vertical alignment.